cai_causal_graph/causal_graph.py

```diff
@@ -1881,8 +1924,10 @@ def get_all_causal_paths(self, source: NodeLike, destination: NodeLike) -> List[
         # but then it would no longer be a DAG as we confirm in the first step of this method.
         if source == destination:
             return []
-
-        return list(networkx.all_simple_paths(self.to_networkx(), source, destination))
+        print('Retrieving networkx graph...')
```

Comment:
Why are you printing here, shouldn't you be logging? Or is this leftover?

Reply:
Leftover sorry, will remove